php/WP_CLI/Runner.php

```diff
@@ -376,31 +376,22 @@ public function find_command_to_run( $args ) {
 			$subcommand = $command->find_subcommand( $args );
 
 			if ( ! $subcommand ) {
-				if ( count( $cmd_path ) > 1 ) {
-					$child       = array_pop( $cmd_path );
-					$parent_name = implode( ' ', $cmd_path );
-					$suggestion  = $this->get_subcommand_suggestion( $child, $command );
 
-					if ( 'network' === $parent_name && 'option' === $child ) {
-						$suggestion = 'meta';
```

Comment:
This special handling for `wp network option` vs `wp network meta` needs to be retained.